Zend/zend_compile.c

```diff
@@ -3862,6 +3863,22 @@ int zend_compile_func_get_args(znode *result, zend_ast_list *args) /* {{{ */
 }
 /* }}} */
 
+int zend_compile_func_array_key_exists(znode *result, zend_ast_list *args) /* {{{ */
+{
+	if (args->children != 2 || args->child[0]->kind == ZEND_AST_UNPACK) {
+		return FAILURE;
+	}
+
+	znode subject, needle;
```

Comment:
Nit: For C99 compatibility these declarations need to be at the top of the function.

Reply:
*C89 =P

Reply:
Updated, although I don't know when this is needed and when it's not, right the next function (zend_compile_func_array_slice) doesn't have declarations on top...

Reply:
For C89 compatibility, it is important that all declarations come before the statements. Since C is block scoped, however, declarations must not necessarily appear at top of the function, but only at top of the block (which appears to be the case for `zend_compile_func_array_slice()`).

Reply:
I see, thanks @cmb69.